src/Inciter/Discretization.C

```diff
@@ -69,23 +73,14 @@ Discretization::Discretization(
   m_nbfac( nbfac ),
   m_bface( bface ),
   m_belem( belem ),
-  m_esuel( tk::genEsuelTet( m_inpoel, 
-                            tk::genEsup(m_inpoel,4) ) ),
-  m_ntfac( tk::genNtfac( 4,
```

Comment:
Magic number (I think I'm just obsessing at this point... :P) ?

Reply:
In the end you are right. We should abstract the 4s and 3s away. Here they mean the number of vertices of the cell. I will put this on my list as that will touch a bunch of other places.

Reply:
In the end you are right. We should abstract the 4s and 3s away. Here they mean the number of vertices of the cell. I will put this on my list as that will touch a bunch of other places.